www/js/theme_editor.js

```diff
@@ -82,14 +151,14 @@ const imageInput = () =>
     h('input.file-input#themebg-upload',
       {
         type: 'file',
-        accept: 'image/png, image/jpeg',
+        accept: '*.png',
```

Comment:
does this do anything? can it be left out if not?

Reply:
Yeah this doesn't do anything. I am still searching a solution to this. So now when we click browse, the file explore dialog box show all kinds of files, I am seeing how can we show only images there. 🙏 If you have anything in mind to solve this, I would be grateful. 🙏 

cc @navdeepsinghkhalsa